lms/static/sass/base/_base.scss

```diff
@@ -183,12 +183,9 @@ span.edx {
 }
 
 .loading-animation {
```

Comment:
Ah, right. It is only the student_account/finish_auth.html that has it. So I added the fa span to the html, I don't know how to check that though.